dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/master/runner/task/CommonTaskProcessor.java

```diff
@@ -70,6 +70,18 @@ public boolean submit(TaskInstance task, ProcessInstance processInstance, int ma
         if (this.taskInstance == null) {
             return false;
         }
+        int taskGroupId = task.getTaskGroupId();
```

Comment:
Is it better to create a new task type for `TGQ`?

Reply:
Do you mean the status of task ?

Reply:
No the task status, I mean the new task type, such as switchTask, sqlTask.

Reply:
Task group is effect on the task of shell、mr、spark and some type of task execute on worker。It's not a node type。